modules/customize-certificates-api-add-default.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/api-server.adoc
+
+[id="add-default-api-server-{context}"]
+= Add an API server default certificate
+
+The default API server certificate is issued by an internal {product-title} 
+cluster CA. This certificate can be replaced so that any API clients, such
+as `oc`, will use the defined certificate.
```

Comment:
The reason to replace the API server cert is so that external clients can verify it, they don't technically "use" the server cert, so this text is incorrect. We should try to convey that in the default case, clients will not be able to verify the server cert. So the purpose of this is to replace the API server certificate by one that is signed by a CA that your external clients have access to in their trust stores. Something like:

"The default API server certificate is issued by an internal {product-title} 
cluster CA. Clients outside of the cluster will not be able to verify the API server's certificate by default.  This certificate can be replaced by one that is issued by a CA that clients trust."

Reply:
This text has been updated to use the recommendation.